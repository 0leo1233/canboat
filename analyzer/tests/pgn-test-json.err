INFO pgn-test [analyzer] Timestamp fixed
INFO pgn-test [analyzer] Detected normal format with all frames on one line
