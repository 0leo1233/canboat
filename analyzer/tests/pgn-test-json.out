{"timestamp":"2011-04-25-06:25:03.603","prio":3,"src":36,"dst":255,"pgn":129029,"description":"GNSS Position Data","fields":{{"SID":230,"bytes"="E6"},{"Date":"2011.04.25","bytes"="F1 3A"},{ "Time": "06:25:12","bytes"="80 9C C6 0D"},{"Latitude":52.7461333,"bytes"="00 12 38 AA 49 EB 51 07"},{"Longitude": 5.1815566,"bytes"="00 0C 44 95 FB 15 B8 00"},{"Altitude":3.400000,"bytes"="40 E1 33 00 00 00 00 00"},{"GNSS type":"GPS+SBAS/WAAS","bytes"="03","bits"="0011"},{"Method":"GNSS fix","bytes"="10","bits"="0001"},{"Integrity":"No integrity checking","bytes"="00","bits"="00"},{{"Number of SVs":9,"bytes"="09"},{"HDOP":0.90,"bytes"="5A 00"},{"PDOP":1.40,"bytes"="8C 00"},{"Geoidal Separation":null,"bytes"="FF FF FF 7F"},{"Reference Stations":0,"bytes"="00"}}}
{"timestamp":"1970-01-01T00:00:00.000Z","prio":3,"src":61,"dst":255,"pgn":127513,"description":"Battery Configuration Status","fields":{{"Instance":0,"bytes"="00"},{"Battery Type":"Gel","bytes"="01","bits"="0001"},{"Supports Equalization":"No","bytes"="00","bits"="00"},{{"Nominal Voltage":"12V","bytes"="01","bits"="0001"},{"Chemistry":"Li","bytes"="10","bits"="0001"},{"Capacity":20,"bytes"="14 00"},{"Temperature Coefficient":2,"bytes"="02"},{"Peukert Exponent":0.002,"bytes"="01"},{"Charge Efficiency Factor":98,"bytes"="62"}}}
{"timestamp":"2022-09-10T12:10:16.614Z","prio":6,"src":5,"dst":255,"pgn":60928,"description":"ISO Address Claim","fields":{{"Unique Number":1088507,"bytes"="FB 9B 10","bits"="111101111110111111011"},{"Manufacturer Code":"Navico","bytes"="60 22","bits"="00100010011"},{"Device Instance Lower":0,"bytes"="00","bits"="000"},{"Device Instance Upper":0,"bytes"="00","bits"="00000"},{"Device Function":"Rudder","bytes"="9B"},{{"Device Class":"Steering and Control surfaces","bytes"="50","bits"="0101000"},{"System Instance":0,"bytes"="00","bits"="0000"},{"Industry Group":"Marine","bytes"="40","bits"="100"},{}}
{"timestamp":"2022-09-10T12:10:16.812Z","prio":6,"src":35,"dst":255,"pgn":60928,"description":"ISO Address Claim","fields":{{"Unique Number":321561,"bytes"="19 E8 04","bits"="001100000110000011001"},{"Manufacturer Code":"Airmar","bytes"="E0 10","bits"="01110000111"},{"Device Instance Lower":0,"bytes"="00","bits"="000"},{"Device Instance Upper":0,"bytes"="00","bits"="00000"},{"Device Function":"Bottom Depth","bytes"="82"},{{"Device Class":"Navigation","bytes"="78","bits"="0111100"},{"System Instance":0,"bytes"="00","bits"="0000"},{"Industry Group":"Marine","bytes"="40","bits"="100"},{}}
{"timestamp":"2016-04-09T16:41:39.628Z","prio":2,"src":16,"dst":255,"pgn":127489,"description":"Engine Parameters, Dynamic","fields":{{"Instance":"Single Engine or Dual Engine Port","bytes"="00"},{"Oil pressure":1.583,"bytes"="2F 06"},{"Oil temperature":null,"bytes"="FF FF"},{"Temperature":23.52,"bytes"="E3 73"},{"Alternator Potential":13.81,"bytes"="65 05"},{"Fuel Rate":null,"bytes"="FF 7F"},{ "Total Engine hours": "01:10:10","bytes"="72 10 00 00"},{"Coolant Pressure":null,"bytes"="FF FF"},{"Fuel Pressure":null,"bytes"="FF FF"},{{"Discrete Status 1": ["Over Temperature","Low Oil Pressure"],"bytes"="06 00"},{"Discrete Status 2":null,"bytes"="00 00"},{"Engine Load":null,"bytes"="7F"},{"Engine Torque":127,"bytes"="7F"}}}
{"timestamp":"1970-01-01T16:41:39.628Z","prio":2,"src":16,"dst":255,"pgn":127489,"description":"Engine Parameters, Dynamic","fields":{{"Instance":"Single Engine or Dual Engine Port","bytes"="00"},{"Oil pressure":1.583,"bytes"="2F 06"},{"Oil temperature":547.65,"bytes"="10 20"},{"Temperature":23.52,"bytes"="E3 73"},{"Alternator Potential":13.81,"bytes"="65 05"},{"Fuel Rate":112.5,"bytes"="65 04"},{ "Total Engine hours": "01:10:10","bytes"="72 10 00 00"},{"Coolant Pressure":8.208,"bytes"="10 20"},{"Fuel Pressure":164.320,"bytes"="30 40"},{{"Discrete Status 1": ["Over Temperature","Low Oil Pressure"],"bytes"="06 00"},{"Discrete Status 2": ["Warning Level 1","Warning Level 2","Power Reduction","Maintenance Needed","Engine Comm Error","Sub or Secondary Throttle","Neutral Start Protect","Engine Shutting Down"],"bytes"="FF 00"},{"Engine Load":48,"bytes"="30"},{"Engine Torque":24,"bytes"="18"}}}
